Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -1589,6 +1589,12 @@ static String mutationObserverNotificationString()
 
 OptionSet<TiledBackingScrollability> UnifiedPDFPlugin::computeScrollability() const
 {
+    if (shouldSizeToFitContent()) {
```

Comment:
Should this path have a default return value of true instead of falling back to evaluating the inner scrollers scrollabolity

Reply:
I don't think the no-FrameView case is super important or even reachable, could go either way